ddp-workspace/projects/ddp-dsm-ui/src/app/tissue/tissue.component.html

```diff
@@ -171,7 +171,7 @@
     </tr>
     <tr>
       <td class="TD--Padding">Pathology Report</td>
-      <td>
+      <td colspan="3">
```

Comment:
I moved the colspan to here to fix a display bug for when tissue was not type block not to change how the tissue page looks like (introduced by another PR of mine, PEPPER-285 )